base/cadvisor/cadvisor.DaemonSet.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  annotations:
+    description: Collects and exports container metrics.
+    seccomp.security.alpha.kubernetes.io/pod: 'docker/default'
+  labels:
+    deploy: sourcegraph
+    sourcegraph-resource-requires: no-cluster-admin
+  name: cadvisor
+spec:
+  selector:
+    matchLabels:
+      name: cadvisor
+  template:
+    metadata:
+      labels:
+        name: cadvisor
+    spec:
+      serviceAccountName: cadvisor
+      containers:
+      - name: cadvisor
+        image: k8s.gcr.io/cadvisor:v0.30.2
```

Comment:
All images here need to be in the form of `index.docker.io/sourcegraph/...:<sourcegraph_version>`

you just need to republish this image, it's really simple just e.g. copy what is done in this directory here: https://github.com/sourcegraph/sourcegraph/blob/master/docker-images/syntax-highlighter/build.sh

You should have a `docker-images/cadvisor/build.sh` and then once that gets merged this line should become:

```
index.docker.io/sourcegraph/cadvisor:insiders@sha256:...
```

And once we do a real release with it, it'll become:

```
index.docker.io/sourcegraph/cadvisor:3.17.0@sha256:...
```

so it's easy to know "is this the right image for my sourcegraph version"

Reply:
Once you do this, can you also update the image in the deploy-sourcegraph-docker repo? we should be doing this there too, must've been missed by accident

Reply:
No, you literally want a file like this in our main repo `docker-images/cadvisor/build.sh`:

```sh
#!/usr/bin/env bash
cd "$(dirname "${BASH_SOURCE[0]}")"
set -ex

# This merely re-tags the image to match our official versioning scheme.
docker pull k8s.gcr.io/cadvisor:v0.30.2@sha256:SOMETHING
docker tag k8s.gcr.io/cadvisor:v0.30.2@sha256:SOMETHING "$IMAGE"
```

you just gotta find the SHA + send that PR to the main repo and you're good

Reply:
ab gotcha, thanks! the example you linked for `syntax-highlighter` seemed to indicate the image being tagged is built in `infrastructure`

Reply:
`deploy-sourcegraph` is at v0.33, latest is v0.36 - should I just bump to the newest one?

Reply:
Yes please update us to the latest version in all locations, should "just work"

Reply:
the example I linked was just the simplest retagging example I could find, it being built in infra is unrelated to the problem here :)

Reply:
ecaff26  and https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/111 :)